templates/entries.html

```diff
@@ -14,18 +14,30 @@
                 {% endfor %}
             {% endif %}
         {% endwith %}
-
+        <div id="nav">
+            <a href="{{ url_for('home') }}"> Home</a>
+            <a href="{{ url_for('users_entries') }}"> Entries</a>
+            <a href="{{ url_for('settings') }}"> Settings</a>
+            <a href="{{ url_for('signout') }}"> Log Out</a>
+        </div>
         <h1>Here are you're previous entries</h1>
-
-
+        
         {% for i in range(0,length) : %}
-            <div class="container p-2 m-5 bg-dark text-white">
-                <form action="/delete_entry" method='POST'>
-                    <button type='submit' name="Delete" value={{user_entries[i].id}}> X </button>
-                </form>
-                <h3>{{user_entries[i].title}}</h3>
-                <p>{{user_entries[i].content}}</p>
+           {% if (i%2)==0  %}
```

Comment:
Originally, the entries would be displayed 1 by 1 all the way down the page, depending on how long it was.
But now with this use of if statements, I was able to create a 2 by 2 layout all the way down the page, which is the correct layout according to our mockup design.